tests/rpmi.at

```diff
@@ -1454,9 +1454,9 @@ RPMDB_INIT
 
 runroot rpmbuild -bb --quiet --define "pkg user" --define "provs %{add_sysuser u myuser 876 - /home/myuser /bin/sh}"\
   /data/SPECS/deptest.spec
-runroot rpm -U /build/RPMS/noarch/deptest-user-1.0-1.noarch.rpm 2> /dev/null
-runroot_other tail -1 /etc/passwd
-runroot rpm -V ${VERIFYOPTS} deptest-user
+runroot rpm -U --root /alt /build/RPMS/noarch/deptest-user-1.0-1.noarch.rpm 2> /dev/null
```

Comment:
For some reason, if (and only if) I run this test locally (that is, not in a CI container), it fails, due to the missing `/alt/etc` directory. I'm not sure what the reason is yet, but you may probably want to add a `mkdir -p ${RPMTEST}/alt/etc` in there.

Reply:
OK, to clarify things a bit, by "locally" above I actually meant "in my own container" :smile: based on a Fedora 37 image. Using the CI container (Fedora 38) worked so I guess the `mkdir` is not needed after all.

I'm not sure why there's a difference between F37 and F38, though, but that probably doesn't matter much for the purposes of this PR.

Reply:
... Thinking a bit further than that, it *does* matter if we ever push this change into F37 or earlier, so... We might as well investigate a bit more... and add that `mkdir` in there :smile:

Reply:
This could potentially be some change in `systemd-sysusers` since it's used in the test to create the user.

Reply:
Yep, this is something I ran into while testing and then ... it went away and I forgot. It apparently went away when I updated to Fedora 38. 

It's easy to reproduce in fresh container: `systemd-sysusers --root /mnt /usr/lib/sysusers.d/basic.conf` in F37 fails with "Failed to take /etc/passwd lock: No such file or directory" but succeeds in F38.

This isn't something that would be backported to older rpms so we don't really care, but it's useful info to keep in mind if others run into this.